LBHackney-IT/lbh-fss-portal-frontend
# Feature/add and submit organisation

**Description**

Add functionality to allow VCSO user to add and submit their organisation

**Test instructions**

- pull this branch 
- navigate to mock-api/account/GET.js and replace with the following content (this will ensure you are logged in as a VCSO user)

```
var mockUsers = require("../mockUsers.json");
var sample = require("lodash/sample");

module.exports = (req, res) => {
  res
    .status(200)
    .json(sample(mockUsers.filter((u) => u.name === "Tommie Dietrich")));
};
``` 

- navigate to **Your organisation** from the main menu, you should be met with instructions to add your organisation, click 'Get started' which will navigate you to **/organisations/add** 
- check form functionality works 
- check against figma designs 

note 1 - you can navigate back to a previous page on the form using the navigation bar, but can only progress forwards by clicking the button at the bottom of the page

note 2 - the designs show the navigation menu as an accordion on mobile, I have added this as a separate task 

Figma designs:
https://www.figma.com/file/t6fanDKvLkCHJsY85ivFTF/Services-Directory?node-id=586%3A8553 
![image](https://user-images.githubusercontent.com/7523692/93495854-7e29a580-f906-11ea-8747-d3348defd7ee.png)

**Tickets**
https://app.clickup.com/t/9e5u80
https://app.clickup.com/t/9e5u76
